beaver/transports/redis_transport.py

```diff
@@ -75,16 +76,27 @@ def callback(self, filename, lines, **kwargs):
             namespace = self._namespace
         self._logger.debug('Got namespace: ' + namespace)
 
+        data_type = self._data_type
+        self._logger.debug('Got data type: ' + data_type)
+
         server = self._get_next_server()
         self._logger.debug('Got redis server: ' + server['url'])
 
         pipeline = server['redis'].pipeline(transaction=False)
 
         for line in lines:
-            pipeline.rpush(
-                namespace,
-                self.format(filename, line, timestamp, **kwargs)
-            )
+            if data_type == 'list':
```

Comment:
Addressed in https://github.com/hectcastro/python-beaver/commit/94812db5e13a95bf20da26bc616e6f12973deff4. Sorry for the original, I wasn't quite sure how to call a method dynamically in Python.